server/profile.py

```diff
@@ -449,6 +449,9 @@ def get_ordered_transcript(profile_uc_dict_list):
             in s.programs and s.id not in closed_scholarship_ids_set]
     scholarships_dict = [s.to_dict() for s in scholarships]
 
+    if profile_user.id != current_user.id:
+        scholarships = []
```

Comment:
Sounds good. I think I was trying to cheat a bit, to avoid the 80 character limit, but should've done it that way originally.